packages/website/pages/manage.js

```diff
@@ -99,6 +99,14 @@ export default function ManageKeys({ user }) {
     </Button>
   )
 
+  /**
+   * @param {string} keyName
+   * @param {number} keyIndex
+   */
+  const getButtonKey = (keyName, keyIndex) => {
```

Comment:
Suggestion (non-blocking):
I don't know if this is particularly valuable, it sorta just indirectly does what string template literals do already.
I feel like it wouldn't be too much to just do this check inline, but i'd leave it to preference.

Reply:
it's used in 4 places so i was just going with reuse here.  i could explode it to just do templating, but this seemed easier to maintain (one place to edit, instead of 4).